### PR TITLE
cups-filters: new package at 1.27.4

### PR DESCRIPTION
**Pull request by Vouivre**
## Description of package

OpenPrinting CUPS filters and backends

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```
/var/db/kiss/installed/cups-filters/version
/var/db/kiss/installed/cups-filters/sources
/var/db/kiss/installed/cups-filters/manifest
/var/db/kiss/installed/cups-filters/etcsums
/var/db/kiss/installed/cups-filters/depends
/var/db/kiss/installed/cups-filters/checksums
/var/db/kiss/installed/cups-filters/build
/var/db/kiss/installed/cups-filters/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/ppd/cupsfilters/pxlmono.ppd
/usr/share/ppd/cupsfilters/pxlcolor.ppd
/usr/share/ppd/cupsfilters/Ricoh-PDF_Printer-PDF.ppd
/usr/share/ppd/cupsfilters/HP-Color_LaserJet_CM3530_MFP-PDF.ppd
/usr/share/ppd/cupsfilters/Generic-PDF_Printer-PDF.ppd
/usr/share/ppd/cupsfilters/Fuji_Xerox-DocuPrint_CM305_df-PDF.ppd
/usr/share/ppd/cupsfilters/
/usr/share/ppd/
/usr/share/man/man8/cups-browsed.8
/usr/share/man/man8/
/usr/share/man/man5/cups-browsed.conf.5
/usr/share/man/man5/
/usr/share/man/man1/foomatic-rip.1
/usr/share/man/man1/driverless.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/doc/cups-filters/README
/usr/share/doc/cups-filters/NEWS
/usr/share/doc/cups-filters/INSTALL
/usr/share/doc/cups-filters/COPYING
/usr/share/doc/cups-filters/AUTHORS
/usr/share/doc/cups-filters/ABOUT-NLS
/usr/share/doc/cups-filters/
/usr/share/doc/
/usr/share/cups/ppdc/pcl.h
/usr/share/cups/ppdc/escp.h
/usr/share/cups/ppdc/
/usr/share/cups/mime/cupsfilters.types
/usr/share/cups/mime/cupsfilters.convs
/usr/share/cups/mime/cupsfilters-poppler.convs
/usr/share/cups/mime/cupsfilters-ghostscript.convs
/usr/share/cups/mime/
/usr/share/cups/drv/cupsfilters.drv
/usr/share/cups/drv/
/usr/share/cups/data/unclassified.pdf
/usr/share/cups/data/topsecret.pdf
/usr/share/cups/data/testprint
/usr/share/cups/data/standard.pdf
/usr/share/cups/data/secret.pdf
/usr/share/cups/data/form_russian_in.odt
/usr/share/cups/data/form_russian.pdf
/usr/share/cups/data/form_english_in.odt
/usr/share/cups/data/form_english.pdf
/usr/share/cups/data/default.pdf
/usr/share/cups/data/default-testpage.pdf
/usr/share/cups/data/confidential.pdf
/usr/share/cups/data/classified.pdf
/usr/share/cups/data/
/usr/share/cups/charsets/pdf.utf-8.simple
/usr/share/cups/charsets/pdf.utf-8.heavy
/usr/share/cups/charsets/pdf.utf-8
/usr/share/cups/charsets/
/usr/share/cups/banners/unclassified
/usr/share/cups/banners/topsecret
/usr/share/cups/banners/standard
/usr/share/cups/banners/secret
/usr/share/cups/banners/form
/usr/share/cups/banners/confidential
/usr/share/cups/banners/classified
/usr/share/cups/banners/
/usr/share/cups/
/usr/share/
/usr/lib/pkgconfig/libfontembed.pc
/usr/lib/pkgconfig/libcupsfilters.pc
/usr/lib/pkgconfig/
/usr/lib/libfontembed.so.1.0.0
/usr/lib/libfontembed.so.1
/usr/lib/libfontembed.so
/usr/lib/libfontembed.la
/usr/lib/libfontembed.a
/usr/lib/libcupsfilters.so.1.0.0
/usr/lib/libcupsfilters.so.1
/usr/lib/libcupsfilters.so
/usr/lib/libcupsfilters.la
/usr/lib/libcupsfilters.a
/usr/lib/cups/filter/texttotext
/usr/lib/cups/filter/texttops
/usr/lib/cups/filter/texttopdf
/usr/lib/cups/filter/sys5ippprinter
/usr/lib/cups/filter/rastertops
/usr/lib/cups/filter/rastertopdf
/usr/lib/cups/filter/rastertopclx
/usr/lib/cups/filter/rastertopclm
/usr/lib/cups/filter/rastertoescpx
/usr/lib/cups/filter/pdftoraster
/usr/lib/cups/filter/pdftops
/usr/lib/cups/filter/pdftopdf
/usr/lib/cups/filter/imagetoraster
/usr/lib/cups/filter/imagetops
/usr/lib/cups/filter/imagetopdf
/usr/lib/cups/filter/gstoraster
/usr/lib/cups/filter/gstopxl
/usr/lib/cups/filter/gstopdf
/usr/lib/cups/filter/foomatic-rip
/usr/lib/cups/filter/commandtopclx
/usr/lib/cups/filter/commandtoescpx
/usr/lib/cups/filter/bannertopdf
/usr/lib/cups/filter/
/usr/lib/cups/driver/driverless
/usr/lib/cups/driver/
/usr/lib/cups/backend/serial
/usr/lib/cups/backend/parallel
/usr/lib/cups/backend/implicitclass
/usr/lib/cups/backend/driverless
/usr/lib/cups/backend/beh
/usr/lib/cups/backend/
/usr/lib/cups/
/usr/lib/
/usr/include/fontembed/sfnt.h
/usr/include/fontembed/iofn.h
/usr/include/fontembed/fontfile.h
/usr/include/fontembed/embed.h
/usr/include/fontembed/bitset.h
/usr/include/fontembed/
/usr/include/cupsfilters/raster.h
/usr/include/cupsfilters/ppdgenerator.h
/usr/include/cupsfilters/pdftoippprinter.h
/usr/include/cupsfilters/ipp.h
/usr/include/cupsfilters/image.h
/usr/include/cupsfilters/driver.h
/usr/include/cupsfilters/colormanager.h
/usr/include/cupsfilters/colord.h
/usr/include/cupsfilters/
/usr/include/
/usr/bin/ttfread
/usr/bin/foomatic-rip
/usr/bin/driverless
/usr/bin/cups-browsed
/usr/bin/
/usr/
/etc/cups/cups-browsed.conf
/etc/cups/
/etc/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/cups-filters/build [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+./configure  \
+    --prefix=/usr \
+    --sysconfdir=/etc \
```
Comment (dylanaraps) [inline community/cups-filters/build:5]: Add `--sbindir=/usr/bin` (There's a file in `sbin`). :+1:

**Comment by dylanaraps** [comment]
Apologies, I forgot about this package before closing the issue. :)

File: community/cups-filters/build (hunk 1 above)
Comment (Vouivre) [reply community/cups-filters/build:6]: Done!

**Comment by Vouivre** [comment]
No problem ;-)

**Pull request merged**